docs/src/pages/components/autocomplete/autocomplete.md

```diff
@@ -116,10 +116,16 @@ It accepts almost the same options as the Autocomplete component minus all the p
 related to the rendering of JSX.
 The Autocomplete component is built on this hook.
 
-```jsx
+```js
 import { useAutocomplete } from '@material-ui/unstyled/AutocompleteUnstyled';
 ```
 
+The `useAutocomplete` hook is also reexported from @material-ui/core for convenience and backwards compatibility.
```

Comment:
I wonder if we need to mention it. We could maybe rely on the implicit rule that all @unstyled is reexported in @core.

```suggestion
The `useAutocomplete` hook is also reexported from @material-ui/core for convenience and backward compatibility.
```

Reply:
In this case, the path is different `@material-ui/core/useAutocomplete` vs `@material-ui/unstyled/AutocompleteUnstyled`, so I think it's worth pointing out.